cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/engine/sql/request/SQLPullRequest.java

```diff
@@ -18,18 +18,21 @@
 
 import io.cdap.cdap.api.annotation.Beta;
 import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.etl.api.engine.sql.dataset.SQLDataset;
+
+import java.io.Serializable;
 
 /**
  * Class representing a Request to pull a dataset from a SQL engine.
  */
 @Beta
-public class SQLPullRequest {
+public class SQLPullRequest implements Serializable {
```

Comment:
Anything that implements Serializable should have serialVersionUID

Reply:
Added the Serial Version UID